.github/workflows/release.yml

```diff
@@ -0,0 +1,45 @@
+name: Release to npm and gh-pages.
+on:
+  # Trigger the workflow on push or pull request,
+  # but only for the master branch
+  push:
+    branches:
+      - master
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    name: A job to execute release
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Install packages.
+        run: yarn install
+      - name: Set up project.
+        run: yarn build
+      - name: Run tests.
+        run: yarn test
+      - name: Release to NPM
+        env:
+          NPM_TOKEN: ${{secrets.NPM_TOKEN}}
+          NPM_EMAIL: ${{secrets.NPM_EMAIL}}
+          NPM_USERNAME: ${{secrets.NPM_USERNAME}}
+        run: yarn release
```

Comment:
As I see this is the first 'destructive' operation. What is the plan if this suceeds, but something later (like the `docs:build` fails)?

Reply:
We would publish it by hand. We could keep a script for that occasion but we can copy the steps from here and do it by hand.